koku/masu/processor/ocp/ocp_cloud_parquet_summary_updater.py

```diff
@@ -111,6 +112,10 @@ def update_aws_summary_tables(self, openshift_provider_uuid, aws_provider_uuid,
                 ocp_accessor.populate_ocp_on_all_project_daily_summary("aws", sql_params)
                 ocp_accessor.populate_ocp_on_all_daily_summary("aws", sql_params)
 
+                ocp_accessor.populate_ui_summary_tables(
```

Comment:
This may be problematic with async processing of AWS and Azure.
I have made changes in #3242 to accommodate this.
See [here](https://github.com/project-koku/koku/pull/3242/files#diff-e1d9f3914577e02d8d6832f80f67d77a97895661105edd473001f6c2e1a8fdf6R2289) and [here](https://github.com/project-koku/koku/pull/3242/files#diff-ee212c79968ef2057866acb49167477a7d600bc073000db65aecfea54028f274R105-R117)

I'm thinking that my ocpall changes should be combined with this.

Reply:
My changes reflect processing for each source throughout #3242